ci_framework/roles/hive/defaults/main.yml

```diff
@@ -0,0 +1,23 @@
+---
+# Copyright Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+
+# All variables intended for modification should be placed in this file.
+# All variables within this role should have a prefix of "cifmw_hive"
+
+cifmw_hive_platform: "openstack"
+cifmw_hive_life_time: 24h
+cifmw_hive_oc_cmd: "oc"
```

Comment:
Readme is saying that defaults to: "oc --kubeconfig {{ cifmw_hive_kubeconfig }}"

Reply:
Changed to : The oc cli command to be used. Defaults to "oc". In the tasks it is set to "oc --kubeconfig {{ cifmw_hive_kubeconfig }}" if cifmw_hive_kubeconfig is defined